src/loop/index.js

```diff
@@ -0,0 +1,36 @@
+// Example: for loop
+for (let i = 1; i <= 5; i++) {
```

Comment:
With for, Please don't use any un-meaning index. you can use
`for (let index = 1; index <=5; index++)`